File: app/src/main/java/com/baro/ui/share/SendReceiveActivity.java
package com.baro.ui.share;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.ImageButton;

import com.baro.R;

public class SendReceiveActivity extends AppCompatActivity {

    private ImageButton sendButton;
    private ImageButton receiveButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_send_receive);

        configureSendButton();
        configureReceiveButton();
    }



    private void configureSendButton() {
        sendButton = findViewById(R.id.btn_send);

        sendButton.setOnClickListener(v -> {
            Intent intentSendActivity = new Intent(SendReceiveActivity.this,
                    BluetoothSendActivity.class);
            startActivity(intentSendActivity);
        });
    }

    private void configureReceiveButton() {
        receiveButton = findViewById(R.id.btn_receive);

        receiveButton.setOnClickListener(v -> {
            Intent intentReceiveActivity = new Intent(SendReceiveActivity.this,
                    BluetoothReceiveActivity.class);
            startActivity(intentReceiveActivity);
        });
    }
}

File: app/src/main/java/com/baro/ui/share/SendReceiveActivity.kt
package com.baro.ui.share

import android.content.Intent
import android.os.Bundle
import android.widget.ImageButton
import androidx.appcompat.app.AppCompatActivity
import com.baro.R
import com.baro.constants.AppCodes
import com.baro.constants.AppTags


class SendReceiveActivity : AppCompatActivity(){


    private lateinit var sendButton: ImageButton
    private lateinit var receiveButton: ImageButton


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_send_receive)
        configureSendButton()
        configureReceiveButton()
    }

    private fun configureSendButton() {
        sendButton = findViewById<ImageButton?>(R.id.btn_send)
        sendButton.setOnClickListener {
            val startWifiDirectActivity = Intent(
                this,
                WifiDirectActivity::class.java)

            startWifiDirectActivity.putExtra(AppTags.WIFIP2P_INTENT.name, AppCodes.WIFIP2P_PEER_SEND.code)
            startActivity(startWifiDirectActivity)
        }
    }
    private fun configureReceiveButton() {
        receiveButton = findViewById<ImageButton?>(R.id.btn_receive)
        receiveButton.setOnClickListener{
            val startWifiDirectActivity = Intent(
                this,
                WifiDirectActivity::class.java)
            startWifiDirectActivity.putExtra(AppTags.WIFIP2P_INTENT.name, AppCodes.WIFIP2P_PEER_RECEIVE.code)
            startActivity(startWifiDirectActivity)
        }

    }

}